extensions/kill-process/src/index.tsx

```diff
@@ -82,8 +93,15 @@ export default function ProcessList() {
   const killProcess = (process: Process) => {
     exec(`kill -9 ${process.id}`);
     setFetchResult(state.filter((p) => p.id !== process.id));
-    clearSearchBar({ forceScrollToTop: true });
-    showHUD(`✅ Killed ${process.processName === "-" ? `process ${process.id}` : process.processName}`);
+    if (multipleKills) {
```

Comment:
i think it is not necessary to have pref for this, I think we can migrate from showHUD and showToast, while window is closed, toast will fallback to showHUD, and we can respect to the preference in "Advanced" for pop to search search.

Reply:
oh, for this change, you are mainly to avoid close main window or clear search bar ?

Reply:
> not necessary to have pref 

My thinking was that people may be currently using the extension to just kill a single rogue process, not killing multiple. So, I don't want their workflow to be impeded

> mainly to avoid close main window or clear search bar ?

I've changed the code a bit to avoid some duplicated logic